### PR TITLE
adds server-config option to ng serve

### PR DESCRIPTION
**Pull request by kdimatteo**
feat (webpack-dev-server): add cli option for static mocks

Add the option to serve static JSON mocks when discreet server is not necessary or available

### CODE REVIEW

**Comment by kdimatteo** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -229,6 +229,26 @@ and then we edit the `package.json` file's start script to be
 
 now run it with `npm start`
 
+### Mocking a Static Backend
+Webpack dev server allows us to modify the ExpressJS `app` object using the `setup` method.
+We do this by passing a file to `--server-config`
+
+We create a file next to the projects `package.json` called `server.conf.js` with the content
```
Comment (victormejia) [inline README.md]: small typo? `project's`

File: tests/e2e/tests/misc/static-server.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { writeFile } from '../../utils/fs';
+import { request } from '../../utils/http';
+import { killAllProcesses, ng } from '../../utils/process';
+import { ngServe } from '../../utils/project';
+import { expectToFail } from '../../utils/utils';
+
+
+export default function() {
+  // const app = express();
```
Comment (victormejia) [inline tests/e2e/tests/misc/static-server.ts]: delete?

**Comment by filipesilva** [comment]
This isn't something we're looking to add. The proxy configuration already let's you do something similar.

**Pull request closed without merging**